src/ansys/pytwin/evaluate/tbrom.py

```diff
@@ -163,17 +174,70 @@ def _reduce_field_input(self, name: str, snapshot: Union[str, Path, np.ndarray])
                 self._infmcs[name][item] = mc[index]
                 index = index + 1
 
+    def project_on_mesh(
+        self, mesh: pv.DataSet, interpolate: bool, named_selection: str = None, progress_bar: bool = False
```

Comment:
Is-it requested to make `TbRom.project_on_mesh` a public method or should we keep internal?

Reply:
1st comment has been considered (also for TwinModel related method) as well as 2nd. 3rd comment needs to be discussed because probably same applied to all the TBROM methods

Reply:
modifications applied to make TBROM methods private, except read/write binary static methods